crates/cargo-test-support/src/compare.rs

```diff
@@ -108,6 +109,20 @@ pub fn assert_ui() -> snapbox::Assert {
         regex::Regex::new("home/\\.cargo/registry/src/-(?<redacted>[a-z0-9]+)").unwrap(),
     )
     .unwrap();
+    subs.insert("[TARGET]", rustc_host()).unwrap();
```

Comment:
I forgot why we separate redactions into UI and e2e. Can/Should we combine them?

Reply:
e2e has more. Maybe when this is done we can reevaluate that but rather not do it during the port.

As for duplicated ones, thats something worth reevaluating at this point.  The first had to have the regex specialized between the two (because i like to keep regexes as simple as possible) and that set the example to follow. We have enough now to be worth reevaluating.

Reply:
The common redactions were merged into one function in #14054 so that `assert_ui` and `assert_e2e` could share them